cloudbuild.yaml

```diff
@@ -74,7 +74,23 @@ steps:
     docker build -t ${_REPOSITORY}/streaming-pipeline:${_TAG} .;
 
 - name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
-  id: 'Run Batch Pipeline JDBC with HAPI source'
+  id: 'Run Batch Pipeline NON_JDBC with HAPI source and HAPI sink'
```

Comment:
Suggestion: drop the `and HAPI sink` from here and everywhere else below; because in most cases we are also writing into Parquet files, which is probably more important than the HAPI sink. Was there a particular reason that you added these which I missed?

Reply:
Dropped